ckanext/scheming/validation.py

```diff
@@ -27,7 +27,7 @@ def scheming_choices(field, schema):
     """
     Require that one of the field choices values is passed.
     """
-    return OneOf([c['value'] for c in field['choices']])
+    return OneOf([c['value'] for c in field['choices']], testValueList=True)
```

Comment:
Oh, looking again that might be spurious. It was intended to get around the problem that really has to be dealt with here: https://github.com/ckan/ckanext-scheming/pull/116/files#diff-db32db2d8eb1d92a5e5f5ded851dd550R55. But here `c['value']`is always a string it seems.